docs/InstallGuide.md

```diff
@@ -0,0 +1,143 @@
+Install Guide
+===============
+
+## Usage instruction for a fresh TorQ Install
+
+
+In your Linux terminal run the following lines copying them in one by one:
+
+    wget https://raw.githubusercontent.com/AquaQAnalytics/TorQ/installtorqapp.sh
+
+    wget --content-disposition https://github.com/AquaQAnalytics/TorQ/archive/3.7.0.tar.gz
+
+    wget --content-disposition https://github.com/AquaQAnalytics/TorQ-Finance-Starter-Pack/archive/v1.9.0.tar.gz
+
+Then to launch the script.
+
+    sh installtorqapp.sh torq=TorQ-3.7.0.tar.gz releasedir=deploy data=datatemp installfile=TorQ-Finance-Starter-Pack-1.9.0.tar.gz env=
+
+Where data parameter and env parameter are optional parameters.
+Full usage of the parameters available in the table below.
+
+The folder structure after installation will look like this:
+
+<center><img src="graphics/Installscript_folder_structure.png" width="600"></center>
+
+Then to run the TorQ stack:
+
+    ./deploy/bin/torq.sh start all
+
+Check if the stack is up 
+
+     ./deploy/bin/torq.sh summary
+
+
+## Parameters used
+
+<table>
+<tr>
+<td> Command line parameter </td> <td> Explanation and usage </td>
+</tr>
+<tr>
+<td> torq </td>
+<td>
+
+
+Is a mandatory parameter that is the full path or relative path to the TorQ installation. It can either be a TorQ Directory where the version is already unzipped, that can be used when multiple TorQ Applications are used on the server for example and all point to a single TorQ main code. This will create a softlink to the relevant TorQ code. Or it can be a .tar.gz file of the TorQ installation for a fresh install. 
+Example usage in the script:
+
+`torq=/home/user/TorQ/TorQ-3.7.0`
+
+Where TorQ-3.7.0 is unzipped directory from the latest release .tar.gz file. Or
+
+`torq=/home/user/TorQ/TorQ-3.7.0.tar.gz` 
+
+Which is the .tar.gz file from GitHub using:
+
+`wget --content-disposition https://github.com/AquaQAnalytics/TorQ/archive/3.7.0.tar.gz`
+
+
+</td>
+</tr>
+<tr>
+<td> releasedir </td>
+<td>
+
+Is a mandatory parameter that is the full path or relative path to the deployment directory that will populate the TorQ and TorQApp. 
+If the directory doesn't exist then script creates one. It can be anything, 
+as per previous AquaQ instructions the folder name has been deploy. 
+The releasedir parameter can be used as follows:
+
+`releasedir=/home/user/deploy`
+
+</td>
+<tr>
+<td> installfile </td>
+<td>
+
+Is a mandatory parameter with the full path or relative path to the TorQApp installation file (ACCEPTS ONLY .tar.gz FILE). 
+Can be used as follows:
+
+`installfile=/home/user/TorQ-FSP/TorQ-Finance-Starter-Pack-master.tar.gz`
+
+</td>
+</tr>
+<tr>
+<td> data </td>
+<td>
+
+An optional parameter. That is if you want to have your database live in a different part of the system rather than the place where the code lives. Can be used as follows:
```

Comment:
Change "database" to "data directory as defined by KDBDATA" (or whatever the variable is called)